R/add_creator_id.R

```diff
@@ -0,0 +1,67 @@
+#' add_creator_id
+#'
+#' This function allows you to add an ORCID or reference ID to a creator in EML.
+#' @param eml EML script to modify
+#' @param orcid ORCID in the format 'https://orcid.org/WWWW-XXXX-YYYY-ZZZZ'
+#' @param id reference ID. Character string to set reference ID for creators with additional roles (i.e. metadataProvider, etc.)
+#' @param surname creator surname (last name), defaults to first creator if not specified. Not case-sensitive.
+#' @keywords eml creator orcid id
+#' @details All prameters other than the EML are optional, but since the point of the function is to modify either the orcid, ref id, or both, you better specify at least one. Requires the crayon package.
+#' @export
+#' @examples
+#' add_creator_id(eml2, orcid = "https://orcid.org/WWWW-XXXX-YYYY-ZZZZ")
```

Comment:
`eml2` isn't defined, it's a bit of pain but the following should work.  You can also just call it `eml` to keep things simple also. 
```
cn <- dataone::CNode('STAGING')
mn <- dataone::getMNode(cnTest,'urn:node:mnTestARCTIC')
eml_pid <- arcticdatautils::create_dummy_metadata(mn)
eml <- EML::read_eml(rawToChar(mn, eml_pid))```